web/zh_cn/collections.textile

```diff
@@ -31,27 +32,44 @@ h1. 基本数据结构
 
 Scala提供了一些不错的集合。
 
-*参考* Effective Scala 对怎样使用 <a href="http://twitter.github.com/effectivescala/#Collections">集合</a>的观点。
+*参考* Effective Scala 对怎样使用<a href="http://twitter.github.com/effectivescala/#Collections">集合</a>的观点。
+
+h2(#Arrays). 数组 Array
+
+数组是有序的，可以包含重复项，并且可变。
+
+<pre>
+scala> val numbers = Array(1, 2, 3, 4, 5, 1, 2, 3, 4, 5)
+numbers: Array[Int] = Array(1, 2, 3, 4, 5, 1, 2, 3, 4, 5)
+
+scala> numbers(3) = 10
```

Comment:
`Array` in English version is OK.
The problem is that there's no `Array` part in the current Chinese version.

So I added it and offered my translation, as well as the original code example. 😄